### DISCUSSION TITLE
People tracking database?

### DISCUSSION THREAD

**Discussion by ravetildon** [ESPresense/ESPresense#228] [Ideas]
Is there a way to save all those beacons from bluetooth, wifi's and phones IMEI info  so I know who comes on my property? I think this would be a great addition to a security system! Or a business like a zoo. So you can track popular animals and what time people like to see animals, how many time they come back to see them on the visit etc. :)

Possibilities are endless! My creative juices are flowing.

**Comment by DTTerastar** [comment]
The randomized MAC addresses make that VERY hard.  Wifi is a bit easier and there is a device that kinda does what you want: https://www.fing.com/products/fingbox
